doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_gs.rst

```diff
@@ -39,7 +39,7 @@ To follow this guide, make sure you have all the required hardware:
 Installing the |NCS|
 ********************
 
-Install the |NCS| following the instructions in the :ref:`install_ncs` documentation page.
+Install the |NCS| following the instructions for both Visual Studio Code and the command line in the :ref:`install_ncs` documentation page.
```

Comment:
```suggestion
Install the |NCS| following the instructions for both |nRFVSC| and the command line in the :ref:`install_ncs` documentation page.
```
We don't support "Visual Studio Code", we support nRF Connect for VS Code, the extension.